### PR TITLE
Add second comment in doc.md README.md

### PR DESCRIPTION
**Pull request by Israel-SanchezH**
Add second comment in doc.md README.md (Homework 3)

### CODE REVIEW

File: README.md
Comment (alexandergmzx) [inline README.md:1]: ok, no real need for the second line, but ok.

File: doc.md
Comment (alexandergmzx) [inline doc.md:1]: Good update, but the file may be better in docs/ directory

**Review by alexandergmzx — APPROVED** [review]
Good work.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 # colaborativo-LED-continental
 
-Este es un ejemplo de como podemos trabajar de manera colaborativa con git.
\ No newline at end of file
+Este es un ejemplo de como podemos trabajar de manera colaborativa con git.
+
+Update 2 (Homework 3)
```
Comment (mocadev1) [inline README.md:5]: What do you mean with Homework 3?

File: doc.md [hunk 2]
```diff
@@ -1 +1,2 @@
 # First Line, First Title
+Update 1 (Homework 3)
```
Comment (mocadev1) [inline doc.md:2]: Homework 3? Again???

**Review by mocadev1 — CHANGES_REQUESTED** [review]
Pls attend the observations

File: README.md (hunk 1 above)
Comment (Israel-SanchezH) [reply README.md:5]: Activity number 3, block two of the training

**Pull request merged**